### DISCUSSION TITLE
v5.3.4

### DISCUSSION THREAD

**Discussion by madeye** [shadowsocks/shadowsocks-android#3168] [General]
Multiple fixes for Android 14 (thanks to @AaronChen0).

## What's Changed
* Hotfix for password checking fail by @dev4u in https://github.com/shadowsocks/shadowsocks-android/pull/3027
* Corrected range for move operation by @MSVhora in https://github.com/shadowsocks/shadowsocks-android/pull/3035
* Add githubusercontent.com to gfwlist by @AaronChen0 in https://github.com/shadowsocks/shadowsocks-android/pull/3039
* Fix base64 check by @AaronChen0 in https://github.com/shadowsocks/shadowsocks-android/pull/3045
* Update shadowsocks-rust by @zedifen in https://github.com/shadowsocks/shadowsocks-android/pull/3119
* fix implicit-function-declaration error when updating NDK to r26+ by @AaronChen0 in https://github.com/shadowsocks/shadowsocks-android/pull/3163
* Target Android 14 by @AaronChen0 in https://github.com/shadowsocks/shadowsocks-android/pull/3165
* Fix a warning from sslocal logcat output by @AaronChen0 in https://github.com/shadowsocks/shadowsocks-android/pull/3166

## New Contributors
* @MSVhora made their first contribution in https://github.com/shadowsocks/shadowsocks-android/pull/3035
* @AaronChen0 made their first contribution in https://github.com/shadowsocks/shadowsocks-android/pull/3039
* @zedifen made their first contribution in https://github.com/shadowsocks/shadowsocks-android/pull/3119

**Full Changelog**: https://github.com/shadowsocks/shadowsocks-android/compare/v5.3.3...v5.3.4

<hr /><em>This discussion was created from the release <a href='https://github.com/shadowsocks/shadowsocks-android/releases/tag/v5.3.4'>v5.3.4</a>.</em>

**Comment by dyhkwong** [comment]
`libimage_processing_util_jni.so` is not 16KB aligned, but it seems that `shadowsocks-android` functions properly on 16KB emulator. AFAIK simply upgrading CamaraX to 1.4.0-rc01 can make it 16KB aligned.

**Comment by AaronChen0** [comment]
~Is the version name `5.3.4-nightly` needed to set to `5.3.4` for release?~
Ok. I see that previous release version names are `nightly` too. Maybe, no need to change.

**Comment by danarieel** [comment]
Hey, maybe I don't understand something, but why `shadowsocks-tv-universal-5.3.4.4.apk` is called that, it's the Android TV version, right? On Google TV the version doesn't work correctly, but the `shadowsocks--universal-5.3.4.apk` works properly, like on default Android. Maybe we should rename the release?

**Comment by mintydev789** [comment]
How come there hasn't been a new version in 8 months?

**Reply by madeye** [reply]
@Mygod what about a new release?

**Reply by Mygod** [reply]
Sure.

**Reply by mintydev789** [reply]
It would be nice to have a new Play Store release too. The current release is so old that the play store displays it as insecure.

**Reply by jackyzy823** [reply]
Hi, a new release would be very very helpful. [This commit](https://github.com/shadowsocks/shadowsocks-android/commit/2bdcb7bb79cb222cc1d609f1ac814e624c67c0f3) is necessary for my device to active service via control center quick button.

Thanks.
